app/javascript/App.jsx

```diff
@@ -32,7 +32,7 @@ export default function App() {
 
   return (
     <>
-      {headerPage && <Header /> }
+      {(headerPage || currentUser.signed_in) && <Header /> }
```

Comment:
Without this - a logged in user wont see the header on the join page

Reply:
Yes forgot this one thanks